src/screens/noAcctHelperFunctions.tsx

```diff
@@ -3,6 +3,12 @@ import * as FS from 'expo-file-system';
 const OFFLINE_STATE_URI = `${FS.documentDirectory}/offline_state.txt`;
 
 export interface OfflineContact {
+  id: string;
+  image: {
+    url: string;
+  };
+  url: string;
+  owner: string;
```

Comment:
Do these fields need to be added to the interface? It seems like for offline contacts, they would have no use.

Reply:
I was getting a bunch of errors before I added them so I think they do.